srcpkgs/hyperrogue/template

```diff
@@ -1,23 +1,38 @@
 # Template file for 'hyperrogue'
 pkgname=hyperrogue
-version=11.3f
+version=12.0u
```

Comment:
12.0z is available

Reply:
I saw the 'z' on itch, but don't really see it in the github tags or releases 🤔

Reply:
well then 🤷‍♀️